src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.OneNote/Main.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+using ManagedCommon;
+using Microsoft.PowerToys.Run.Plugin.OneNote.Properties;
+using ScipBe.Common.Office.OneNote;
+using Windows.Win32;
+using Windows.Win32.Foundation;
+using Windows.Win32.UI.WindowsAndMessaging;
+using Wox.Plugin;
+
+namespace Microsoft.PowerToys.Run.Plugin.OneNote
+{
+    /// <summary>
+    /// A power launcher plugin to search across time zones.
+    /// </summary>
+    public class Main : IPlugin, IPluginI18n
+    {
+        /// <summary>
+        /// A value indicating if the OneNote interop library was able to successfully initialize.
+        /// </summary>
+        private readonly bool _oneNoteInstalled;
+
+        /// <summary>
+        /// The initial context for this plugin (contains API and meta-data)
+        /// </summary>
+        private PluginInitContext? _context;
+
+        /// <summary>
+        /// The path to the icon for each result
+        /// </summary>
+        private string _iconPath;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Main"/> class.
+        /// </summary>
+        public Main()
+        {
+            UpdateIconPath(Theme.Light);
```

Comment:
Why do we need this if we update the icon theme in the init method based on the theme?

Reply:
This is just to ensure that the non-nullable _iconPath field has a value before exiting the constructor, otherwise we'd get a warning. I don't have access to the _context here yet, so I have to just pick a theme value, which will then be overwritten when Init is called